.github/workflows/Tests.yml

```
@@ -58,3 +64,24 @@
+    
+    - name: Create Dusk CI Environment
+      run: |
+        cp .env.dusk.local .env.dusk.ci
+        echo "APP_URL=http://127.0.0.1:8000" >> .env.dusk.ci
+        echo "DUSK_DRIVER_URL=http://selenium:4444/wd/hub" >> .env.dusk.ci
+    
+    - name: Start Laravel Server
+      run: php artisan serve --host=127.0.0.1 --port=8000 &
+      env:
+        APP_ENV: dusk.ci
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)